docs/docs/index.md

```diff
@@ -4,27 +4,12 @@ ms.date: 09/30/2019
 ---
 
 # Factory Orchestrator
+Factory Orchestrator, is a cross-platform service for organizing, executing, and logging a set of executable scripts, binaries, or ["Tasks"](tasks-and-tasklists.md) on a system. Factory Orchestrator tracks task information, including run unique per-run results and logging; even persisting task state to allow the service to be resilient to data loss due to system failure. Factory Orchestrator also provides an optional [UWP client app](use-the-factory-orchestrator-app.md) and [robust client API surface](use-the-factory-orchestrator-api.md) for clients to monitor & interact with the service via the App or C# .NET, C# UWP, or PowerShell code. The UWP app and and any other client can communicate with any Factory Orchestrator service running on the same system and/or over a network!

```

Comment:
>  is a cross-platform service for organizing, 

does service mean the same to everyone?

could we replace with 'toolset'? or 'tooling'?

Reply:
changed to "system service" which based on my searching seems pretty prolific.